pkg/actors/timers/timers.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/dapr/dapr/pkg/actors/internal"
 	diag "github.com/dapr/dapr/pkg/diagnostics"
-	"github.com/dapr/kit/eventqueue"
+	"github.com/dapr/kit/events/queue"
```

Comment:
Is it an independent change? If so, please post another PR since this one is too big now.

Reply:
No, this PR requires [this](https://github.com/dapr/kit/pull/58) kit change which created the subpackage `events`.